public/app/features/explore/Logs/LogsMetaRow.tsx

```diff
@@ -58,6 +49,14 @@ export const LogsMetaRow = memo(
   }: Props) => {
     const style = useStyles2(getStyles);
 
+    // Filter out default fields from displayedFields to show only user-added fields
+    const nonDefaultFields = useMemo(() => {
+      if (!displayedFields?.length || !defaultDisplayedFields?.length) {
```

Comment:
I think I'm just getting thrown by the name of the `nonDefaultFields` variable, but if we don't have default displayed fields, but we have displayed fields, aren't those non-default fields?